index.js

```diff
@@ -90,11 +90,19 @@ var createBrowserStackClient = function (/* config.browserStack */config) {
 
   config = config || {}
 
-  // TODO(vojta): handle no username/pwd
-  var client = api.createClient({
+  var options = {
     username: env.BROWSER_STACK_USERNAME || config.username,
     password: env.BROWSER_STACK_ACCESS_KEY || config.accessKey
-  })
+  }
+
+  if (config.proxyHost && config.proxyPort) {
+    var proxyAuth = (config.proxyUser && config.proxyPass)
+      ? (config.proxyUser + ':' + config.proxyPass + '@') : ''
+    options.proxy = 'http://' + proxyAuth + config.proxyHost + ':' + config.proxyPort
```

Comment:
Yes that would be good, but it should default to `http` when not set. 

Also can you please add some details about this to the readme so people know it's there :)